dap_prinz_green_jobs/pipeline/ojo_application/ojo_sample/deduplication.py

```diff
@@ -0,0 +1,69 @@
+"""
+A script to deduplicate the OJO dataset
+
+"""
+
+import pandas as pd
+import pyarrow.parquet as pq
+
+from tqdm import tqdm
+
+from dap_prinz_green_jobs.getters.data_getters import save_to_s3, get_s3_resource
+from dap_prinz_green_jobs import BUCKET_NAME, logger, config
+from dap_prinz_green_jobs.pipeline.ojo_application.ojo_sample.deduplication_utils import (
+    short_hash,
+    get_deduplicated_job_adverts,
+)
+
+s3 = get_s3_resource()
```

Comment:
part of me thinks we put this in data_getters.py so we aren't constantly re-creating this object all over scripts in the codebase

Reply:
I've done it! I've made the relevant changes across the repo here [f06b807](https://github.com/nestauk/dap_prinz_green_jobs/pull/19/commits/f06b8076eb211f923a3af71148d28e7960cc5635) just need to remember to not add the s3 argument to the load and save functions now!